Formula/p/proto.rb

```diff
@@ -0,0 +1,48 @@
+class Proto < Formula
+  desc "Pluggable multi-language version manager"
+  homepage "https://moonrepo.dev/proto"
+  url "https://github.com/moonrepo/proto/archive/refs/tags/v0.30.0.tar.gz"
+  sha256 "a564e4c8a4d2483b19e02bd1b3cdac5db4ad61cc0e2d217203f0a641897ba7f4"
+  license "MIT"
+  head "https://github.com/moonrepo/proto.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "pkg-config" => :build
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "crates/cli")
+    generate_completions_from_executable(bin/"proto", "completions", "--shell")
+
+    bin.each_child do |f|
+      basename = f.basename
+
+      (libexec/"bin").install f
+      (bin/basename).write_env_script libexec/"bin"/basename, PROTO_INSTALL_DIR: opt_prefix/"bin"
+    end
+  end
+
+  def caveats
+    <<~EOS
+      To finish the installation, run:
+        proto setup
+    EOS
+  end
+
```

Comment:
```suggestion
```

Reply:
`caveats` are only for things specific to Homebrew packaging, so we should not include this.

Reply:
`proto setup` is usually called by their install script automatically, it's purpose is to add PROTO_HOME env var, as well as add the shims and bin folders to the PATH.

Alternatively I can replace it with:

```
 To use proto, add the following line (or equivalent) to your shell profile
      e.g. ~/.profile or ~/.zshrc:
        export PROTO_HOME="$HOME/.proto"
        export PATH="$PROTO_HOME/shims:$PROTO_HOME/bin:$PATH"
  Restart your terminal for the settings to take effect.
```

If this doesn't change your opinion, then I'll remove it as requested. :)

Reply:
If the standard install handles this automatically, then I think it's a good reason to have the caveat for Homebrew. Thanks for the explanation, and we can keep it as is.